File: hw/villagomez/u2/hw18ZooJS/ec.edu.espe.zoo.model/Cage.js
class Cage {

    constructor(id, type){
        this._id = id;
        this._type = type;
    }

    toString() {
        return `Cage{id=${this._id}, type=${this._type}}`;
    }

    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get type() {
        return this._type;
    }

    set type(type) {
        this._type = type;
    }

}

module.exports = Cage;

File: ws/Perez/U2/WS24/Zoo/src/main/java/ec/edu/espe/zoo/model/Cage.java
package ec.edu.espe.zoo.model;

/**
 *
 * @author Carlos Perez,The Javas, DCCO-ESPE
 * @version 1.0.0
 */
public class Cage {
    private int id;
    private String type;

    @Override
    public String toString() {
        return "Cage{" + "id=" + id + ", type=" + type + '}';
    }

    public Cage(int id, String type) {
        this.id = id;
        this.type = type;
    }

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the type
     */
    public String getType() {
        return type;
    }

    /**
     * @param type the type to set
     */
    public void setType(String type) {
        this.type = type;
    }
    
}
